mlflow/genai/judges/prompts/user_frustration.py

```diff
@@ -0,0 +1,66 @@
+# NB: User-facing name for the user frustration assessment.
+USER_FRUSTRATION_ASSESSMENT_NAME = "user_frustration"
+
+USER_FRUSTRATION_PROMPT = """\
+You are an expert evaluator of human-AI conversations.
+Your job is to determine whether the user is frustrated because of the AI assistant.
+
+You must base your judgment only on evidence in the conversation.
+
+## Instructions
+A. Frustration specifically caused by the AI (count this)
+The user is considered frustrated because of the AI if ANY of the following appear:
+1. Explicit frustration directed at the AI
+Examples (not exhaustive):
+- Complaints about the AI's behavior or performance ("You're not helping," "Why do you keep messing this up?")
+- Irritated or hostile tone toward the AI
+- User giving up because of the AI's mistakes ("Forget it," "This is useless" after the AI's wrong answer)
+
+2. Implicit frustration directed at the AI
+Signals that the AI is failing and the user is reacting to that failure:
+The user shows irritation through behavior, not wording.
+Look for patterns, not isolated turns.
+Count as frustration if you see two or more of the following AI-caused signals:
+- Repeated Corrections
+    - User must correct the AI's mistake or misunderstanding more than once.
+    - ("That's not what I asked," "No, I meant X," "Not that, the other thing.")
```

Comment:
The format of examples in each of these signals is a little different, can we be consistent to avoid confusion?